_projects/2023/100506380/100506380.Rmd

```diff
@@ -0,0 +1,480 @@
+---
+title: "The Phenomenon of Poverty"
+description: |
+  Visualizing the global distribution of disposable income between 1990 and 2019- Our World in Data.
+categories: "2023"
+author: "Frederick Peña Sims"
+date: "2023-12-19"
```

Comment:
Please use ```date: "`r Sys.Date()`"``` instead. The date will be automatically generated when the document is knitted.